### ISSUE TITLE
Realtime parametrization and configuration

### ISSUE THREAD

**Issue by antoniodeluca** [antoniodeluca/dn2a#28]
Implement methods that allow the user code to change in realtime the parametrization and configuration of the neural networks and relative environments. Should be possible, for example, to change the learning rate in realtime while the training is happening.

**Comment by antoniodeluca** [comment]
Removing from issues as this work is going to be described and planned through different approach.
